guides/common/modules/proc_enabling-rh-cloud-and-insights-client-reports-on-hosts.adoc

```diff
@@ -0,0 +1,19 @@
+[id="enabling-rh-cloud-and-insights-client-reports-on-hosts_{context}"]
+= Enabling rh-cloud and insights-client reports on hosts
+
+You can set the `host_registration_insights` parameter to *True* to enable _rh-cloud_ and _insights-client_ reports.
+{Project} will create _rh-cloud_ reports and _insights-client_ will upload reports to the cloud.
+
+Use the following procedure to change the value of `host_registration_insights` parameter.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Host* > *Provisioning Setup* > *Operating Systems*.
+. Select any {RHEL} operating systems for which you want to change the value.
+. On the *Parameters* tab, click on the edit button of `host_registration_insights`.
+. Set the value to *True*.
+
+.Additional resources
+* You can set this parameter at any level.
+For more information, see {ProvisioningDocURL}Host_Parameter_Hierarchy_provisioning[Host parameter hierarchy] in _{ProvisioningDocTitle}_.
++
+However, if you set the parameter on any non-{RHEL} operating systems, it automatically uploads new reports.
```

Comment:
I'd say that adding this line means that lines 16-19 can no longer be considered additional resources. An additional resource is just a link. However, line 19 adds an explanation of product behavior that would be better suited for the `.Procedure` part.